Documentation/PageTsconfig/Mod/WebList.rst

```diff
@@ -200,6 +200,62 @@ Example: Hide the column selector
 
     mod.web_list.displayColumnSelector = 0
 
+..  _pageTsConfigWebList-downloadPresets:
+
+downloadPresets
+===============
+
+..  confval:: downloadPresets.[table]
+    :name: mod-web-list-downloadPresets
+    :type: array of presets
+
+    ..  versionadded:: 13.2
+        A new feature has been introduced, adding the ability to pick the exported
+        data columns from a list of configurable presets.
+
+    This property adds presets of preselected fields to the download area in
```

Comment:
In this case I think `fields` is OK, as the visible content of the fields of the TCEform will be used for the export, and not the plain value from database table.